### PR TITLE
[Fix]: Searchbar popups and Add popups in leaderboard

### PR DESCRIPTION
**Pull request by abesheknarayan**

### CODE REVIEW

**Comment by ishanisri** [comment]
Add tooltip for stocks in search bar even when the user does not open it, and try placing the $ symbol at the same place.

File: public/src/components/leaderboard/Leaderboard.tsx [hunk 1]
```diff
@@ -84,12 +84,13 @@ export class Leaderboard extends React.Component<LeaderboardProps, LeaderboardSt
         const state = this.state;
 
         const leaderboardEntries = state.leaderboardEntries.map((entry, index) => (
-            <tr key={index}>
+            <tr key={index} className={entry.getIsblocked()?"leaderboard-blocked": ""}>
```
Comment (ishanisri) [inline public/src/components/leaderboard/Leaderboard.tsx]: Check class=`disabled` for this.

Comment (abesheknarayan) [reply public/src/components/leaderboard/Leaderboard.tsx]: Popups are prohibited for disabled so I wrote css for that

File: public/src/components/leaderboard/Leaderboard.tsx [hunk 2]
```diff
@@ -84,12 +84,13 @@ export class Leaderboard extends React.Component<LeaderboardProps, LeaderboardSt
...
                 <td><strong>{entry.getRank()}</strong></td>
-                <td><strong>{entry.getUserName()}</strong></td>
+                {entry.getIsblocked() && <td className="box" data-tooltip="Account Blocked for a day" data-position="left center" data-variation="mini" ><strong>{entry.getUserName()}  </strong> </td> }
+                {!entry.getIsblocked() && <td><strong>{entry.getUserName()}</strong></td>}
                 <td><strong>{addCommas(entry.getCash())}</strong></td>
-                <td className={entry.getStockWorth() >= 0 ? "green" : "red"}><strong>{addCommas(entry.getStockWorth())}</strong></td>
-                <td className="green"><strong>{addCommas(entry.getTotalWorth())}</strong></td>
+                <td className={!entry.getIsblocked()?(entry.getStockWorth() >= 0 ? "green" : "red"): (entry.getStockWorth() >= 0 ? "leaderboard-blocked-cell-green": "leaderboard-blocked-cell-red")}><strong>{addCommas(entry.getStockWorth())}</strong></td>
+                <td className={!entry.getIsblocked()?"green": "leaderboard-blocked-cell-green"}><strong>{addCommas(entry.getTotalWorth())}</strong></td>
```
Comment (nimishagarwal76) [inline public/src/components/leaderboard/Leaderboard.tsx]: Write code according to proto, it is getIsBlocked()

**Pull request merged**